index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harshika's Portfolio</title>
+    <link rel="stylesheet" href="style.css" class="css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+</head>
+<body>
+    <div class="mobileinfo">
+        <h1>Welcome</h1>
+          </div>
+          <div class="mobilemenu">
+          <i class="fa-sharp fa-solid fa-bars"></i>  </div>
```

Comment:
- [ ] Kindly wrap your hamburger button in a `<nav>` tag.The `nav` tag is for semantics reason